Code/GraphMol/ChemReactions/Reaction.cpp

```diff
@@ -211,11 +211,14 @@ bool ChemicalReaction::validate(unsigned int &numWarnings,
                qIter != query->endChildren(); ++qIter) {
             queries.push_back((*qIter).get());
           }
-          if (query->getDescription() == "AtomFormalCharge") {
+          if (query->getDescription() == "AtomFormalCharge" ||
+              query->getDescription() == "AtomNegativeFormalCharge") {
             int qval;
+            int neg =
+                query->getDescription() == "NegativeFormalCharge" ? -1 : 1;
```

Comment:
Shouldn't this be == "AtomNegativeFormalCharge"?  I can't imagine that this should ever be true.

Reply:
Yep. Will fix it

Reply:
Yep. Will fix it